server/cmwell-ws/app/logic/CRUDServiceFS.scala

```diff
@@ -462,6 +469,19 @@ class CRUDServiceFS @Inject()(implicit ec: ExecutionContext, sys: ActorSystem) e
     }
   }
 
+  def consumeKafka(topic: String, partition: Int, offset: Long, maxLengthOpt: Option[Long]): Source[Array[Byte], Consumer.Control] = {
+    val byteArrayDeserializer = new ByteArrayDeserializer()
+    val subscription = Subscriptions.assignmentWithOffset(new TopicPartition(topic, partition) -> offset)
+    val consumerSettings = ConsumerSettings(Grid.system, byteArrayDeserializer, byteArrayDeserializer)
+      .withBootstrapServers("localhost:9092")
```

Comment:
You shouldn't hard-code the bootstrap servers (and in a clustered environment it should have 3 and not just one)

Reply:
So if this will happen to reach a WS on one of non “seed nodes”, will it fail? I’m going to have to wire that in cons... :/